nservicebus/azure-service-bus/error-handling.md

```diff
@@ -0,0 +1,50 @@
+
+#### Broker side exceptions
+
+A subset of these exceptions originates from within the Azure Service Bus service itself. These exceptions can be recognized by the `TrackingId` field and its value in the exception message. Usually, these exceptions are also transient in nature and will disappear after a few minutes. 
+
+Note: If broker side exceptions persist the `TrackingId` value can be used to contact Azure Support to investigate further with Microsoft and Azure Service Bus teams.
+
+#### Message Size problems
+
+A peculiar behavior of the Azure Service Bus SDK is how it reports on actual message size. It can only do so accurately after a message has been sent. Before sending the reported message size only covers the body section and not the final size that would include header and serialization overhead. This can obviously lead to unexpected results when trying to send a message.
+
+If the application is intended to send large messages, it should always leverage the [NServiceBus Databus](/nservicebus/messaging/databus/) to send large payloads. In case the application sends regular messages that borderline with the maximum message size than extra precaution may be necessary.
+
+The transport deals with this problem for a large part by performing an [estimated batch size calculation](batching.md#batching-messages-sent-from-a-handler-padding-and-estimated-batch-size-calculation) that includes both body and headers as well as a percentage for padding due to serialization. 
+
+Even with this calculation in place, there is no guarantee that a message will eventually fit the limits after serialization. And if it doesn't then the Azure Service Bus client will throw a `MessageSizeExceededException`. The transport will catch this exception and invoke an instance of `IHandleOversizedBrokeredMessages`, which has a default implementation that throws a `MessageTooLargeException` suggesting to use the DataBus feature. A [custom implementation](oversized-sends.md) for handling of oversized sends can be provided.
+
+### Dead Letter Queue
+
+Next to the client-side error handling concerns described above, the broker itself also has several error handling capabilities:
+* When the client fails to process a message and exceeds a maximum number of attempts.
+* When message expires (optional).
+* When subscription filters do not match (optional).
+
+Broker will respond to any of these conditions by moving the message to a dead letter queue, which is a sub-queue below a messaging entity. Each regular queue or subscription has a dead letter queue.
+
+From a management perspective, this behavior is not ideal, though, as it requires operations to check multiple dead letter queue instances for failed messages. One solution could be to set up [dead letter queue forwarding](dlq-forwarding.md) to a centralized error or dead letter queue.
```

Comment:
>From a management perspective, this behavior is not ideal, though, as it requires operations to check multiple dead letter queue instances for failed messages.

has too many commas. How about:

From a management perspective, this behavior is not ideal as it requires operations to check multiple dead letter queue instances for failed messages.